robocop/checkers/misc.py

```diff
@@ -299,17 +306,18 @@
 
         For example::
 
-        Example Keyword
```

Comment:
I didn't change the docs here, only added indent - if the code example has the same indent as ``line::``, it will not render correctly:

![image](https://user-images.githubusercontent.com/8532066/233602267-68690e8e-721f-4aec-accd-7db7be443e4c.png)